### PR TITLE
Add seed support to faux reader

### PR DESCRIPTION
**Pull request by abellgithub**
Also remove broken normal/uniform number generation.

### CODE REVIEW

File: pdal/util/Utils.cpp [hunk 1]
```diff
@@ -81,25 +81,6 @@ double Utils::random(double minimum, double maximum)
 }
 
 
-double Utils::uniform(const double& minimum, const double& maximum,
```
Comment (hobu) [inline pdal/util/Utils.cpp:84]: Do any filters still use this? I suppose this is pre-c++11 stuff.

 It seems out of scope to remove these capabilities in this PR, or the description should be updated to reflect we're removing them.

Comment (abellgithub) [reply pdal/util/Utils.cpp:84]: They were only used by filters.faux and they were broken, because you had to set a seed for each call, which was dumb.  I probably wrote them.
